src/hardware/gus.cpp

```diff
@@ -1365,67 +1373,67 @@ void Gus::WriteToRegister()
 	default:
 		break;
 		// If the above weren't triggered, then fall-through
-		// to the voice-specific switch below.
+		// to the target_voice-specific switch below.
 	}
 
-	// All the registers below here involve voices
-	if (!voice)
+	// All the registers below operated on the target voice
+	if (!target_voice)
```

Comment:
I agree. 
In this case, I really wanted to use `voice` in the multiple places where we iterator over `voices` (instead of simply `v`), so I had to move this to accommodate. 

 I also think is a beneficial change, because before this it was simply "voice" and didn't give evidence that this "voice" was actually being selected or targets by the DOS application.  Now it clearer - the DOS application actual targets a specific voice to work on via the register calls.